bm_inventory_r4pac21.json

```diff
@@ -0,0 +1,1445 @@
+{
+    "nodes": [
+      {
+        "name": "MOC-R4PAC21U37-S1A-OBM",
+        "properties": {
+          "capabilities": "iscsi_boot:True"
+        },
+        "resource_class": "baremetal",
+        "driver": "ipmi",
+        "driver_info": {
+          "deploy_kernel": "42298c88-7927-4fa5-999a-a4f03a8c3272",
+          "deploy_ramdisk": "bb5adbc2-0d7a-4993-ace4-b30a6385c9fd",
+          "ipmi_username": "root",
+          "ipmi_password": "",
+          "ipmi_address": "10.2.10.191",
+          "ipmi_terminal_port": 10191
+        },
+        "ports": [
+          {
+            "address": "40:5C:FD:67:E9:41",
+            "physical_network": "datacentre",
+            "local_link_connection": {
+              "switch_info": "MOC-R4PAC21-SW-TORS-A",
+              "port_id": "tengigabitethernet 1/15/1",
+              "switch_id": "d8:9e:f3:a4:46:a2"
+            }
+          },
+          {
+            "address": "40:5C:FD:67:E9:44",
+            "physical_network": "datacentre",
+            "local_link_connection": {
+              "switch_info": "MOC-R4PAC21-SW-TORS-B",
```

Comment:
Ah, so each node has a port managed by two separate switches - is the intent that the user know which one they want to manipulate?

Reply:
Yes, the user can use one or both ports. The switches themselves are redundant so the best way for redundancy would be a port channel across both ports, but I don't think that's something ESI can setup automatically right now (correct me if I'm wrong). That would involve making a port channel and defining `vlt-peer-link` on both sides.

Reply:
Okay! Just wanted to be sure, because by default users don't usually specify a baremetal port, and Ironic just grabs one at random. But it sounds like that behavior is fine, correct?

Reply:
Yeah that should be okay, either port will work, as long as the user can use both if they specify.